test/integration/user_can_see_past_orders_test.rb

```diff
@@ -56,7 +54,7 @@ class UserCanSeePastOrdersTest < ActionDispatch::IntegrationTest
       assert page.has_content?("Hiking the Alps 1 (Travellers: 1)")
       assert page.has_content?("Hiking the Alps 1 (Travellers: 2)")
       assert page.has_content?("$3,003")
-      assert page.has_content?("#{Time.now.strftime("%B %d, %Y")}")
+      assert page.has_content?("#{Order.first.created_at.strftime("%B %d, %Y")}")
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings inside interpolations.